operator/operator.make

```diff
@@ -12,3 +12,19 @@ _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION):
 # GOROOT is needed because of https://github.com/operator-framework/operator-sdk/issues/1854#issuecomment-525132306
 operator-generate-k8s: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION)
 	GOROOT=$(shell $(GO) env GOROOT) _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) generate k8s
+
+operator-generate-crds: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION)
+	GOROOT=$(shell $(GO) env GOROOT) $< generate crds
+
+operator-clean-crds:
+	rm -rf ./deploy/crds
+
+RELEASE_VERSION=0.7.0
+operator-generate-csv: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) operator-generate-crds
```

Comment:
When I run "make operator-generate-csv", I end up with changes:
```
$ git diff
diff --git a/deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml b/deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml
index 76c6659b..e9020d93 100644
--- a/deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml
+++ b/deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml
@@ -2,23 +2,7 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: |-
-      [
-        {
-          "apiVersion": "pmem-csi.intel.com/v1alpha1",
-          "kind": "Deployment",
-          "metadata": {
-            "name": "pmem-csi.intel.com"
-          },
-          "spec": {
-            "deviceMode": "lvm",
-            "nodeSelector": {
-              "storage": "pmem"
-            },
-            "pmemPercentage": 50
-          }
-        }
-      ]
+    alm-examples: '[]'
     capabilities: Basic Install
     categories: Storage
     certified: "false"
@@ -208,6 +192,14 @@ spec:
               volumes:
               - emptyDir: {}
                 name: tmp
+      - name: pmem-csi.intel.com
+        spec:
+          selector: null
+          strategy: {}
+          template:
+            metadata: {}
+            spec:
+              containers: null
       permissions:
       - rules:
         - apiGroups:
```

Reply:
To generate `alm-examples` we need [this](https://github.com/operator-framework/operator-sdk/pull/3334) fix.
I generated this CSV file after applying the fix on operator-sdk v0.18.2 release branch.